paper.md

```diff
@@ -48,72 +48,60 @@ used in a wide range of contexts including change point detection [@JMLR:v24:22-
-specialists, and professionals. An important use case is as a front-end to working with other
-languages, for which automated tooling providing scaffolding / boilerplate code for integrating external languages
-are widely used. For example, embedding C++ code is a good way to resolve performance bottlenecks
-within R packages, and it can be easily accomplished using cpp11 [@cpp11] and Rcpp [@rcpp_jss].
-Rust has similar performance to C++ but also has other beneficial features such as declarative memory management,
-providing compile-time guarantees for memory safety without the need for a garbage collector.
+<!--Java?-->
+
+The strength of R is its ecosystem of packages, the vast majority of which are available from [CRAN](https://cran.r-project.org).
+They are primarily written by research scientists, specialists, and professionals. 
+Another important use case of R packages is being a front-end for other languages.
+Automated toolings that provide scaffolding and boilerplate code are widely used to simplify cross-language integration.
+For example, embedding C++ code is a good way to resolve performance bottlenecks within R packages, and it can be easily accomplished using cpp11 [@cpp11] or Rcpp [@rcpp_jss].
+Rust demonstrates similar performance to C++, but it also offers other beneficial features such as declarative memory management, and compile-time guarantees for memory safety in the absence of a garbage collector.
 <!-- Rust and R also share support for WebAssembly [webR](https://docs.r-wasm.org/webr/latest/). -->
-We note that other scientific computing communities
-have introduced plug-ins for Rust, including Python via [PyO3](https://github.com/PyO3/pyo3),
+We note that other scientific computing communities have already introduced plug-ins for Rust, including Python via [PyO3](https://github.com/PyO3/pyo3),
 and Julia via [jlrs](https://github.com/Taaitaaiger/jlrs).
-However, to the authors' knowledge there is currently no fully featured mechanism for integrating Rust within an R package
-in a similar manner to that provided by Rcpp for C++.
-
-This paper introduces a collection of four Rust crates and an R package that collectively make up the extendr project.
-The goal of this project is to provide (automatic) binding of Rust to R, using an opinionated and
-ergonomics-focused suite of tools that facilitate the use of Rust code within R packages.
-This is achieved by providing an emulation of the R data model within Rust, integration
-of Rust tooling in the R-package build systems, a rust developer experience in
-R, and functions for preparing publishing of Rust-powered R-packages to CRAN.
-A webpage with an overview of the extendr crates/packages and access to comprehensive
-API documentation is provided at [extendr.github.io](https://extendr.github.io/).
+However, to the authors' knowledge, there currently is no fully featured mechanism for integrating Rust within an R package like that provided by Rcpp for C++.
```

Comment:
@Ilia-Kosenkov do you accept this? Otherwise I think these corrections are fine, and should be merged. I'll use the arxiv reference for the roxido thing.

Reply:
Yep, please proceed.